src/main/java/de/focusshift/zeiterfassung/feedback/FeedbackGivenListenerEmail.java

```diff
@@ -3,19 +3,20 @@
 import de.focusshift.zeiterfassung.email.EMailConstants;
 import de.focusshift.zeiterfassung.email.EMailService;
 import de.focusshift.zeiterfassung.tenancy.user.EMailAddress;
+import jakarta.mail.MessagingException;
 import org.slf4j.Logger;
 import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.event.EventListener;
 import org.springframework.scheduling.annotation.Async;
 import org.springframework.stereotype.Component;
 import org.thymeleaf.ITemplateEngine;
 import org.thymeleaf.context.Context;
 
-import jakarta.mail.MessagingException;
-
 import static java.lang.invoke.MethodHandles.lookup;
 import static org.slf4j.LoggerFactory.getLogger;
 
+@ConditionalOnProperty(prefix = "zeiterfassung.feedback", name = "enabled", havingValue = "true")
 @Component
 class FeedbackGivenListenerEmail {
```

Comment:
Diese Beans könnte man nun auch in der FeedbackConfiguration herausgeben, falls das property passt. Dann hat man alles an einem Fleck.